### DISCUSSION TITLE
Encoding an intentionally left gap

### DISCUSSION THREAD

**Discussion by MarcinKrawczuk** [SChAth/ScAthiop#1644] [Q&A] [answered]
What is a proper way to encode an intentionally left gap i. e. a gap which was to be filled in later, as it seems to be the case in the note which I attach?
![gap](https://user-images.githubusercontent.com/62760230/102511628-4ce97e80-4089-11eb-9efb-60511d921a67.JPG)

@thea-m @eu-genia

**Comment by PietroLiuzzo** [comment, marked as answer]
the element to be used in such case is [`<space>`](https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-space.html) In our Guidelines we offer a specific value in case we know that this was left for a word to be rubricated. see https://betamasaheft.eu/Guidelines/?id=space

**Reply by MarcinKrawczuk** [reply]
That is exactly what I needed, thank you, but it seems that neither of the two values (i. e. rubrication and owner) really fits here. Is there a possibility to mark it as "unspecified", "unknown" or something like that?

**Reply by PietroLiuzzo** [reply]
I think you can use it without any value for `@reason`, or, because it is  a open list of allowed values, any other value you like. I would go for no attribute, since it is optional.

**Reply by MarcinKrawczuk** [reply]
OK, this ends the discussion, as far as I'm concerned.

**Reply by PietroLiuzzo** [reply]
could you please mark it as answered? thanks a lot!

**Reply by MarcinKrawczuk** [reply]
Please tell me how, I don't think I've seen this board before...

**Reply by PietroLiuzzo** [reply]
no worries :) this is very new! it is the circle with the tick at the top of this box on the right, where I first commented.
